buildroot-external/board/pc/ova/hassos-hook.sh

```diff
@@ -32,7 +32,7 @@ function hassos_post_image() {
 
     cp -a "${BOARD_DIR}/home-assistant.ovf" "${OVA_DATA}/home-assistant.ovf"
     qemu-img convert -O vmdk -o subformat=streamOptimized "${HDD_IMG}" "${OVA_DATA}/home-assistant.vmdk"
-    (cd "${OVA_DATA}" || exit 1; sha256sum --tag home-assistant.* >home-assistant.mf)
+    (cd "${OVA_DATA}" || exit 1; openssl sha256 home-assistant.* >home-assistant.mf)
```

Comment:
IMHO this should better be

```
"${HOST_DIR}/bin/openssl" sha256
```

so that the openssl version build during the buildroot run is used and not the openssl binary installed in the system itself.